src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -603,6 +611,8 @@ export default {
 		this.loadingPage = basketItems.some(item => item.__typename === 'LoanReservation'); // eslint-disable-line no-underscore-dangle, max-len
 	},
 	mounted() {
+		this.$root.$on('handleAddToBasket', this.handleAddToBasket);
+		this.$root.$on('jumpToLoans', this.jumpToLoans);
```

Comment:
Are these necessary?

Reply:
jumpToLoans is necessary to be an event handler because the event handler that the button in the hero section triggers is specified in contentful for DynamicHeroClassic.
Add to basket isn't really necessary for this PR though, I can remove it.  I imagine I might need that event for when I swap the loan cards out for the classic loan cards but I haven't really gotten there yet so it'll be cleaner to just add it in that PR when I get to it.

Reply:
Ahh yes, makes sense now. Shouldn't be a problem to keep them both here

Reply:
In other contexts, we manage these events at a component level and turn them off on unmount... I think this is ok  to just activate in this context for now b/c folks generally do not leave this page which might leave these events active.

Reply:
@mcstover Where/how are these events triggered?

Reply:
@michelleinez please also add a beforeDestroy hook to remove these events when the component is destroyed. See `MonthlyGoodSelectorDesktop.vue` for a good example of this

Reply:
@dyersituations This comes from contentful. There is an option to add a web event name to a button in contentful, in this case you would add the `jumpToLoans` as the value

Reply:
added the hook